### PR TITLE
feat(remote-label): display label on schedule if it's given from remote

### PR DESCRIPTION
**Pull request by mattwang44**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
A field of `ProposedTalkEvent` named `is_remote` is add. If a talk event is assigned as remote, a label will be displayed on the schedule page (after a new version of the schedule is generated).

## Steps to Test This Pull Request
1. Go to the admin page and select specific talks as remote (check the tickbox and save).
2. Go to `<host>/<lang>/conference/schedule/new/`

## Expected behavior
The label indicating the talk is given from remote will be displayed.

## Related Issue
#908 

## More Information

image

## Flaw

- Cannot translate for the text on the label
- If the title of the talk is too long, the label will be hidden
- Alternatives: 
    1. put on an icon along with other icons (e.g. python level) instead of a label?
	2. change the text on the label from `Remote Talk` to `Remote`?

### CODE REVIEW

File: src/events/templatetags/events.py [hunk 1]
```diff
@@ -123,7 +123,8 @@ def get_talk_event_display(event):
             proposal.additionalspeaker_set
             .values_list('user__speaker_name', flat=True),
         )
-    return _render_talk_event_template(event, proposal, speaker_names, False)
+    is_remote = event.is_remote
+    return _render_talk_event_template(event, proposal, speaker_names, False, is_remote)
```
Comment (yychen) [inline src/events/templatetags/events.py:127]: why don't you
```suggestion
    return _render_talk_event_template(event, proposal, speaker_names, False, event.is_remote)
```

**Review by yychen — COMMENTED** [review]
I was thinking about adding something in the icon
圖片

But your work will do as well, since I think remote is also a big deal, your way is more explicit.
However, you might also need to make the same label inside the talk detail page (eg. when you click the talk on the schedule, that page should also have the remote flag shown)

File: src/events/templatetags/events.py (hunk 1 above)
Comment (mattwang44) [reply src/events/templatetags/events.py]: fixed!

**Comment by mattwang44** [comment]
New patch is committed. Here is what it looks like:
image

**Review by yychen — APPROVED** [review]
LGTM. I think the style should can be adjusted a little bit. But let's just roll this out first and then see if we have time to do that.

**Pull request merged**